recirq/quantum_chess/quantum_board_test.py

```diff
@@ -452,9 +452,8 @@ def test_slide_with_two_path_qubits_coherence():
         u.squares_to_bitboard(['d1', 'e2', 'g1']): 1 / 2,
     })
 
-@pytest.mark.parametrize('board', BIG_CIRQ_BOARDS)
```

Comment:
Why are some boards removed? The point of the optimizations is to make more circuits lay out successfully; if they are having the opposite effect then we should find a different way to do it.